src/openrct2-ui/windows/ServerList.cpp

```diff
@@ -396,7 +397,9 @@ static void window_server_list_paint(rct_window *w, rct_drawpixelinfo *dpi)
     const char * version = NETWORK_STREAM_ID;
     gfx_draw_string_left(dpi, STR_NETWORK_VERSION, (void*)&version, COLOUR_WHITE, w->x + 324, w->y + w->widgets[WIDX_START_SERVER].top);
 
-    gfx_draw_string_left(dpi, STR_X_PLAYERS_ONLINE, (void*)&_numPlayersOnline, COLOUR_WHITE, w->x + 8, w->y + w->height - 15);
+    if (status_text != STR_EMPTY) {
```

Comment:
can it ever be `STR_EMPTY`? It doesn't look like it from the changes here.

Reply:
True.